Applications/Tizen_CAPI/Tizen_CAPI_config.ini

```diff
@@ -0,0 +1,45 @@
+# Network Section : Network
+[Network]
+Type = NeuralNetwork	# Network Type : Regression, KNN, NeuralNetwork
+Layers = inputlayer \
+	 outputlayer	#Layers of Neuralnetwork
+Learning_rate = 0.0001 	# Learning Rate
+Decay_rate = 0.96	# for the decay_rate for the decayed learning rate
+Decay_steps = 1000       # decay step for the exponential decayed learning rate
+Epoch = 10		# Epoch
+Optimizer = adam 	# Optimizer : sgd (stochastic gradien decent),
+ 	    		#             adam (Adamtive Moment Estimation)
+Cost = cross  		# Cost(loss) function : msr (mean square root error)
+                        #                       cross ( for cross entropy )
+Weight_Decay = l2norm
+weight_Decay_Lambda = 0.005
+Model = "model.bin"  	# model path to save / read
+minibatch = 32		# mini batch size
+beta1 = 0.9 		# beta 1 for adam
+beta2 = 0.9999	# beta 2 for adam
+epsilon = 1e-7	# epsilon for adam
+
+[DataSet]
+BufferSize=100
+TrainData="trainingSet.dat"
+ValidData="trainingSet.dat"
```

Comment:
Same name, intended?

Reply:
For unittesting, this should be fine, but it'd be better if separated.

Reply:
Yes. like @myungjoo mentioned, it is intended. I will find the way to do it better. may be make separate package.

Reply:
Hmm, then, is `valSet.dat` not used?

Reply:
yes. valSet.dat is not used. this is just test case to evaluate the training mechanism. If we use training data as validation, this should go nearly 100% accuracy even though it is relatively small data. That means training mechanism is Ok at least. This example is intended to check this.

Reply:
Thanks for the clear explanation!